### PR TITLE
Swagger

### PR DESCRIPTION
**Pull request by LiorAgnin**
## Proposed changes

Describe the big picture of your changes here to communicate to the maintainers why we should accept this pull request.

If it fixes a bug or resolves a feature request, be sure to link to that issue.

## Types of changes

What types of changes does your code introduce?
_Put an `x` in the boxes that apply_

- [ ] Bugfix (non-breaking change which fixes an issue)
- [ ] Refactor or a modification (non-breaking change)
- [x] New feature (non-breaking change which adds functionality)

### CODE REVIEW

File: apps/charge-accounts-service/src/operators/operators.controller.ts [hunk 1]
```diff
@@ -7,7 +7,11 @@ import { AuthOperatorDto } from '@app/accounts-service/operators/dto/auth-operat
 import { Response } from 'express'
 import { WebhookEvent } from '@app/apps-service/payments/interfaces/webhook-event.interface'
 import { MessagePattern } from '@nestjs/microservices'
+import { ApiBody, ApiCreatedResponse, ApiNotFoundResponse, ApiOperation, ApiParam, ApiTags, ApiUnauthorizedResponse } from '@nestjs/swagger'
+import { AuthOperator } from '@app/accounts-service/operators/entities/auth-operator.entity'
+import { CreateOperatorUser } from '@app/accounts-service/operators/entities/create-operator-user.entity'
 
+@ApiTags('Operators')
```
Comment (MusabShakeel576) [inline apps/charge-accounts-service/src/operators/operators.controller.ts:14]: @LiorAgnin Operators is not available in Swagger UI on port 5002, is it visible to you?

Comment (LiorAgnin) [reply apps/charge-accounts-service/src/operators/operators.controller.ts:14]: Oh is that so. I'll check. Thanks. @MusabShakeel576

Comment (LiorAgnin) [reply apps/charge-accounts-service/src/operators/operators.controller.ts:14]: @MusabShakeel576 The issue is that the Operators are not used in the API service. I've set up a dedicated swagger module for the accounts service to address this. To see it. Just run the backend and go to http://localhost:5001/api#/

Comment (MusabShakeel576) [reply apps/charge-accounts-service/src/operators/operators.controller.ts:14]: @LiorAgnin port 5001 is not working, there is no Swagger UI.

Comment (LiorAgnin) [reply apps/charge-accounts-service/src/operators/operators.controller.ts:14]: @MusabShakeel576  I double checked and it works for me. Can you try again?

Comment (MusabShakeel576) [reply apps/charge-accounts-service/src/operators/operators.controller.ts:14]: @LiorAgnin sure, I'll try again

Comment (MusabShakeel576) [reply apps/charge-accounts-service/src/operators/operators.controller.ts:14]: @LiorAgnin it's working, the issue was, AMPLITUDE_API_KEY env was not set. Maybe we can update the env example file.

**Review by MusabShakeel576 — APPROVED** [review]
LGTM

**Pull request merged**